SchoolHousing/SchoolHousing/ElementarySchoolSchema.json

```diff
@@ -0,0 +1,167 @@
+﻿{
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "type": "array",
+  "items": {
+    "$ref": "#/definitions/ElementarySchoolElement"
+  },
+  "definitions": {
+    "ElementarySchoolElement": {
+      "type": "object",
+      "additionalProperties": true,
+      "properties": {
+        "school_id": {
+          "type": "string",
+          "format": "integer"
+        },
+        "name_of_school": {
+          "type": "string"
+        },
+        "street_address": {
+          "type": "string"
+        },
+        "city": {
+          "type": "string"
+        },
+        "state": {
+          "type": "string"
+        },
+        "zip_code": {
+          "type": "string",
+          "format": "integer"
+        },
+        "phone_number": {
+          "type": "string"
+        },
+        "website": {
+          "type": "string",
+          "format": "uri",
+          "qt-uri-protocols": [
+            "http"
+          ]
+        },
+        "cps_performance_policy_level": {
+          "type": "string"
```

Comment:
As long as you are adding a schema, you might as well add validation rules as well.  How many characters can be in a string?  What is the maximum value of a number?

Reply:
This is noted